tests/flytekit/unit/common_tests/test_helpers.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import absolute_import
+
+from os import path as _path
+
+from flyteidl.core import workflow_pb2 as _workflow_pb2
+
+from flytekit.models.core import workflow as _workflow_model
+
```

Comment:
any tests?

also, I can't comment on the bin files, but maybe move to `resources/protos/`? Just in case other things come in

Reply:
forgot to delete this file.  sure, will move.